### PR TITLE
Fixing gRPC round robin for placement service

### PR DESCRIPTION
**Pull request by elena-kolevska**
Related to actors e2e tests failing

References:
- https://github.com/grpc/grpc/blob/master/doc/environment_variables.md
- https://github.com/grpc/grpc/blob/master/doc/load-balancing.md

### CODE REVIEW

**Review by artursouza — COMMENTED** [review]
No concerns from my side. Waiting on Cassie's approval to merge.

File: pkg/actors/placement/client.go [hunk 1]
```diff
@@ -104,20 +117,24 @@ var noop = func() {}
...
 	c.streamConnectedCond.L.Lock()
 	if !c.streamConnAlive {
 		c.streamConnectedCond.Broadcast()
 		c.streamConnectedCond.L.Unlock()
 		return
 	}
 
-	c.drain(c.clientStream, c.clientConn)
+	if closeConnection {
+		c.drain(c.clientStream, c.clientConn)
```
Comment (cicoyle) [inline pkg/actors/placement/client.go:134]: I think we can get rid of the bool here and conditional, and do the nil check in the drain func like you already have

Comment (cicoyle) [reply pkg/actors/placement/client.go:134]: Elena mentioned this is intentional to reuse the same client conn for grpc load balancing. this is fine then 👍🏻

**Review by cicoyle — APPROVED** [review]
LGTM assuming tests are passing consistently now :)

**Review by artursouza — APPROVED** [review]

**Pull request merged**